Sources/SpeziLLM/Views/LLMChatView.swift

```diff
@@ -92,9 +95,13 @@ public struct LLMChatView<Session: LLMSession>: View {
     /// Creates a ``LLMChatView`` with a `Binding` of a ``LLMSession`` that provides developers with a basic chat view to interact with a Spezi LLM.
     ///
     /// - Parameters:
-    ///   - model: A `Binding` of a  ``LLMSession`` that contains the ready-to-use LLM to generate outputs based on user input.
-    public init(session: Binding<Session>) {
+    ///   - session: A `Binding` of a  ``LLMSession`` that contains the ready-to-use LLM to generate outputs based on user input.
```

Comment:
Thanks for fixing the docs here (so switching `model` to `session`), greatly appreciated!